docs/source/quickstart/advanced.rst

```diff
@@ -3,11 +3,52 @@
 Advanced Usage
 ==============
 
+* :ref:`run-multiple-accounts`
 * :ref:`run-multiple-regions`
 * :ref:`report-multiple-regions`
 * :ref:`report-custom-fields`
 * :ref:`policy_resource_limits`
 
+.. _run-multiple-accounts:
+
+Running against multiple accounts
+---------------------------------
+
+Support for running against multiple AWS accounts, Azure subscriptions, and GCP
+projects can be achieved through c7n-org, which must be installed on top 
+of Cloud Custodian::
+
+  $ pip install c7n-org
+
+C7n-org allows users to run Custodian policies against multiple accounts,
+run a script against multiple AWS accounts, or report on AWS cross account policy execution.
```

Comment:
lot of aws specific words in here, these statements are all true across providers.

you can drop most of this content, and just link internally to the tools/c7n-org.md which is part of the main docs site now.